release/gcb/cloud_builder.sh

```diff
@@ -44,7 +44,7 @@ GOPATH="$(cd "$ROOT/../../.." && pwd)"
 export GOPATH
 echo gopath is "$GOPATH"
 # this is needed for istioctl and other parts of build to get the version info
-export ISTIO_VERSION="${CB_VERSION}"
+export ISTIO_VERSION="${CB_VERSION:-master}"
```

Comment:
I suspect istio version should never be "master".

Reply:
I needed to add this because the shell_check was throwing an error without it.  Apparently as soon as I added a CNI_VERSION parameter it wasn't happy that this wasn't initialized.   Should I change to a invalid value?   Alternatively if I change the name of CNI_VERSION then I might also avoid the shell_check failure.

Reply:
Do you need a separate CNI_VERSION? Why should it be different from ISTIO_VERSION in the release?